pkg/oci/utils.go

```diff
@@ -587,6 +587,19 @@ func ContainerConfig(ocispec CompatOCISpec, bundlePath, cid, console string, det
 		return vc.ContainerConfig{}, err
 	}
 
+	var resources vc.ContainerResources
+	resources.Resources, err = vmConfig(ocispec, RuntimeConfig{VMConfig: resources.Resources})
```

Comment:
This is not needed. The computation translating cpu quota and cpu period into vCPUs should be done inside virtcontainers directly. Those are Linux generic metrics, no need to translate them before to provide them as container resources.